app/test_main.py

```diff
@@ -1 +1,42 @@
-# write your code here
+from typing import Callable
+
+from unittest import mock
+
+import pytest
+
+from app.main import can_access_google_page
+
+
+@mock.patch("app.main.valid_google_url")
+@mock.patch("app.main.has_internet_connection")
+@pytest.mark.parametrize(
+    "has_internet_connection, valid_google_url, expected",
+    [
+        pytest.param(
+            True, True, "Accessible",
+            id="Should return Accessible"
+        ),
+        pytest.param(
+            False, False, "Not accessible",
+            id="Should return Not accessible if at least one False"
+        ),
+        pytest.param(
+            False, True, "Not accessible",
+            id="Should return Not accessible if at least one False"
+        ),
+        pytest.param(
+            True, False, "Not accessible",
+            id="Should return Not accessible if at least one False"
```

Comment:
id must be unique for each test, but you have three identical

Reply:
Done)